common/changes/@itwin/core-quantity/rob-bearingNorthDefault_2024-09-25-10-42.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-quantity",
+      "comment": "Change default azimuth/bearing base from east to north",
```

Comment:
can we have a callout for this in nextver.md?

Reply:
done